packages/page-accounts/src/Accounts/index.spec.ts

```diff
@@ -102,6 +107,98 @@ describe('Accounts page', () => {
 
       expect(row2TotalActual).toHaveTextContent(row2TotalExpected);
     });
+
+    it('displays some summary', () => {
+      accountsPage.renderPage([
+        {
+          address: '5DAAnrj7VHTznn2AWBemMuyBwZWs6FNFjdyVXUeYum3PTXFy',
+          balance: {
+            freeBalance: balance(500)
+          }
+        },
+        {
+          address: '5HGjWAeFDfFCWPsjFQdVV2Msvz2XtMktvgocEZcCj68kUMaw',
+          balance: {
+            freeBalance: balance(200),
+            reservedBalance: balance(150)
+          }
+        }
+      ]);
+
+      const summaries = screen.queryAllByTestId(/card-summary:total \w+/i);
+
+      expect(summaries).not.toHaveLength(0);
+    });
+
+    it('displays balance summary', async () => {
+      accountsPage.renderPage(defaultStakingAccounts);
+
+      // Sum of Free + Reserved for all accounts
+      const expectedAmount = balance(850);
+      const expectedText = accountsPage.format(expectedAmount);
+
+      const summary = await screen.findByTestId(/card-summary:(total )?balance/i);
+
+      expect(summary).toHaveTextContent(expectedText);
+    });
+
+    it('displays transferrable summary', async () => {
+      accountsPage.renderPage(defaultStakingAccounts);
+
+      // Sum of AvailableBalances for all accounts
+      const expectedAmount = balance(1000);
+      const expectedText = accountsPage.format(expectedAmount);
+
+      const summary = await screen.findByTestId(/card-summary:(total )?transferrable/i);
+
+      expect(summary).toHaveTextContent(expectedText);
+    });
+
+    it('displays locked summary', async () => {
+      accountsPage.renderPage(defaultStakingAccounts);
+
+      const expectedAmount = balance(820);
+      const expectedText = accountsPage.format(expectedAmount);
+
+      const summary = await screen.findByTestId(/card-summary:(total )?locked/i);
+
+      expect(summary).toHaveTextContent(expectedText);
+    });
+
+    it('displays bonded summary', async () => {
+      accountsPage.renderPage(defaultStakingAccounts);
+
+      // Sum of stakingLedger.active.unwrap() for all accounts
+      const expectedAmount = balance(90);
+      const expectedText = accountsPage.format(expectedAmount);
+
+      const summary = await screen.findByTestId(/card-summary:(total )?bonded/i);
+
+      expect(summary).toHaveTextContent(expectedText);
+    });
+
+    it('displays unbonding summary', async () => {
+      accountsPage.renderPage(defaultStakingAccounts);
+
+      // Sum of "unlocking" for each account
```

Comment:
Most of the tests have explaining comments, but not all. I guess we can remove all comments, as the logic is pretty simple. No strong opinion, though.

My preferred way would be to see all relevant data (and only relevant data) in each test case. Eg. if we sum only the "unlocking" part, we render with `merge(accountsWithStaking, [{unlocking: 500}, {unlocking:1000}])`. But I understand that could be too bit too much for some of the cases we have here.

Reply:
Oh yeah, this will look great

Reply:
Updated as per comment. I think it is an improvement overall

Reply:
Yup, I agree though we need to be careful with those opinionated default shortcuts, as if we put to much into them in a rigid manner, then it may be hard later to unwind defaults to have e.g. account addresses + a portion of default facilities in order to easily code unit tests for sorting features. Just FYI as we cannot know it upfront and be prepared. Just a word of caution.

Great suggestion + realization :) Thanks!